### PR TITLE
dshekhar/bfd 265 v2 coverage rfc

### PR DESCRIPTION
**Pull request by dshekhar18**
### Change Details

RFC for outlining the BFD team's approach for migrating the Coverage Resource to FHIR R4 / CARIN BB as part of BFD v2 migration efforts.  This RFC is specific to Coverage FHIR resource. Coverage v2 RFC is located at:

[Coverage v2 RFC.md](https://github.com/CMSgov/beneficiary-fhir-data/blob/dshekhar/BFD-265-v2_Coverage_RFC/rfcs/BFDv2/Coverage%20v2%20RFC.md)

### Acceptance Validation

All data and design changes which are needed in Coverage v2 data model are documented.  Current state and Future state are depicted with differential views, some POC codes along with payloads. Any unresolved questions are clearly highlighted.

### Feedback Requested

Make sure all design changes are clearly outlined for  FHIR STU3 (BFDv1) migration to FHIR R4 (BFDv2) for the Coverage FHIR resource.
### External References

- BFD-264 (https://jira.cms.gov/browse/BFD-265)

### Security Implications

- [ ] new software dependencies
No

- [ ] altered security controls
 No

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

Testing scenarios:
NA

### CODE REVIEW

**Review by keithdadkins — DISMISSED** [review]
RFC with no code changes. Approved.

File: rfcs/BFDv2/BFD_v1_Patient-Current_State.txt [hunk 1]
```diff
@@ -0,0 +1,134 @@
+{
```
Comment (jzulim) [inline rfcs/BFDv2/BFD_v1_Patient-Current_State.txt:1]: I would remove this file, as it does not pertain to the coverage RFC.

File: rfcs/BFDv2/BFD_v2_Patient-Future_State.txt [hunk 2]
```diff
@@ -0,0 +1,172 @@
+{
```
Comment (jzulim) [inline rfcs/BFDv2/BFD_v2_Patient-Future_State.txt:1]: I would remove this file as it doesnt pertain to the coverage RFC.

File: rfcs/BFDv2/BFDv2 RFC.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
+# RFC Proposal
```
Comment (jzulim) [inline rfcs/BFDv2/BFDv2 RFC.md:1]: Does this file belong here?

File: rfcs/BFDv2/Coverage v2 RFC.md [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+
+Type of class such as 'group' or 'plan' (CodeableConcept). E.g. Medicare.  
+
+Coverage.class.value  
+
+Value associated with the type (String). E.g “Part D”.  
+
+##### Additional CARIN MANDATORY New Fields to be Added:
+
+**Subscriber Id:**
```
Comment (jzulim) [inline rfcs/BFDv2/Coverage v2 RFC.md:137]: What field will we use here? MBI, HICN, bene id?

File: rfcs/BFDv2/Coverage v2 RFC.md [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+[Differential_view_3](./Differential_view_Coveragev2_3.PNG)
+
+
+### Unresolved Questions
+[Unresolved Questions]: #unresolved-questions
+
+
+The following questions need to be resolved prior to merging this RFC:
+
+1. Can raw un-hashed MBI be used a Subscriber Id?
```
Comment (jzulim) [inline rfcs/BFDv2/Coverage v2 RFC.md:269]: Did we land on a solution for this in previous conversations?

File: rfcs/BFDv2/Coverage v2 RFC.md [hunk 6]
```diff
@@ -0,0 +1,312 @@
...
+
+
+2. What code for Coverage “type” can be used  from this Code System:
+
+http://terminology.hl7.org/CodeSystem/v3-ActCode
```
Comment (jzulim) [inline rfcs/BFDv2/Coverage v2 RFC.md:274]: Feels like a product question, anything I can do to help with this?

File: rfcs/BFDv2/Patient RFC.md [hunk 7]
```diff
@@ -0,0 +1,250 @@
+# RFC Proposal
```
Comment (jzulim) [inline rfcs/BFDv2/Patient RFC.md:1]: Should this RFC be removed?

**Comment by jzulim** [comment]
Overall, great info. I think we should consider removing the patient related material from this PR. There's also some open product questions we might need input on before closing this loop. I read up on slicing, pretty cool! Thanks for putting this together.

**Comment by dshekhar18** [comment]
PR closed.

**Pull request closed without merging**